app/app.py

```diff
@@ -13,13 +14,20 @@ def question_router(question_type, question_id):
 
     return question.json, status
 
+@app.route(API_VERSION + "/question/<question_type>/<question_id>", methods=["POST"])
+def get_solution(question_type, question_id):
+
+    attempt = request.get_json(force=True)
```

Comment:
What does the `force=True` do?

Reply:
it forces the request to be parsed as json, i.e. ignores mimetype. Definitely on the todo list to start using proper HttpResponse objects rather than bonestock json

Reply:
Sick